itest/itest_sourceipmap.sh

```diff
@@ -11,7 +11,7 @@ function cleanup {
     set +e
     kill $MAPGEN_PID
     docker kill $CONTAINER_NAME
-    rm $FIFONAME
+    rm -f $FIFONAME
```

Comment:
I'm guessing without the `-f` we left this dangling for some reason? If figured into the itest it wouldn't be necessary to do this if the docker image is recreated each time